localization/french/culture/wc_cultures_l_french.yml

```
@@ -1,416 +1,570 @@
+
+
+ gurubashi:0 "Gurubashi"
+ gurubashi_collective_noun:1 "Gurubashi"
+ amani:0 "Amani"
+ amani_collective_noun:1 "Amani"
+ drakkari:0 "Drakkari"
+ drakkari_collective_noun:1 "Drakkari"
+ farraki:0 "Farraki"
+ farraki_collective_noun:1 "Farraki"
+ zandalari:0 "Zandalari"
+ zandalari_collective_noun:1 "Zandalari"
+ dark_troll:0 "Troll sombre"
+ dark_troll_collective_noun:1 "Trolls sombres"
+ blood_troll:0 "Nazmani"
+ blood_troll_collective_noun:1 "Nazmani"
 
  #Tauren group
- tauren_group:0 "yaungole"
- tauren_group_collective_noun:1 "yaungols"
- taunka:0 "taunka"
- taunka_collective_noun:1 "taunkas"
- tauren:0 "taurène"
- tauren_collective_noun:1 "taurens"
- highmountain:0 "haut-roc"
- highmountain_collective_noun:1 "haut-roc"
- yaungol:0 "yaungole"
- yaungol_collective_noun:1 "yaungole"
+
+
+ taunka:0 "Taunka"
+ taunka_collective_noun:1 "Taunkas"
+ tauren:0 "Taurène"
+ tauren_collective_noun:1 "Taurens"
+ highmountain:0 "Haut-roc"
+ highmountain_collective_noun:1 "Haut-roc"
+ yaungol:0 "Yaungole"
+ yaungol_collective_noun:1 "Yaungole"
 
  #Vrykul group
- vrykul_group:0 "vrykule"
- vrykul_group_collective_noun:1 "vrykuls"
- vrykul:0 "fjorthir"
- vrykul_collective_noun:1 "fjorthirs"
- kvaldir:0 "kvaldir"
- kvaldir_collective_noun:1 "kvaldirs"
- hyldnir:0 "hyldnir"
- hyldnir_collective_noun:1 "hyldnirs"
- frost_vrykul:0 "iskronir"
- frost_vrykul_collective_noun:1 "iskronirs"
- drust:0 "drust"
- drust_collective_noun:1 "drusts"
- stormheimir:0 "tornheimir"
- stormheimir_collective_noun:1 "tornheimirs"
+
+
+ vrykul:0 "Fjorthir"
+ vrykul_collective_noun:1 "Fjorthirs"
+ kvaldir:0 "Kvaldir"
+ kvaldir_collective_noun:1 "Kvaldirs"
+ hyldnir:0 "Hyldnir"
+ hyldnir_collective_noun:1 "Hyldnirs"
+ frost_vrykul:0 "Iskronir"
+ frost_vrykul_collective_noun:1 "Iskronirs"
+ drust:0 "Drust"
+ drust_collective_noun:1 "Drusts"
+ stormheimir:0 "Tornheimir"
+ stormheimir_collective_noun:1 "Tornheimirs"
+ lucardir:0 "Lucardir"
+ lucardir_collective_noun:1 "Lucardir"
 
  #Aqir group
- aqir_group:0 "aqir"
- aqir_group_collective_noun:1 "aqir"
- qiraji:0 "qiraji"
- qiraji_collective_noun:1 "qiraji"
- nerubian:0 "nérubienne"
- nerubian_collective_noun:1 "nérubiens"
- mantid:0 "mantide"
- mantid_collective_noun:1 "mantides"
+
+
+ qiraji:0 "Qiraji"
+ qiraji_collective_noun:1 "Qiraji"
+ nerubian:0 "Nérubienne"
+ nerubian_collective_noun:1 "Nérubiens"
+ mantid:0 "Mantide"
+ mantid_collective_noun:1 "Mantides"
 
  #Dragon group
- dragon_group:0 "draconique"
- dragon_group_collective_noun:1 "dragons"
- green_dragon:0 "draconique verte"
- green_dragon_collective_noun:1 "dragons verts"
- red_dragon:0 "draconique rouge"
- red_dragon_collective_noun:1 "dragons rouges"
- blue_dragon:0 "draconique bleue"
- blue_dragon_collective_noun:1 "dragons bleus"
- black_dragon:0 "draconique noire"
- black_dragon_collective_noun:1 "dragons noirs"
- bronze_dragon:0 "draconique de bronze"
- bronze_dragon_collective_noun:1 "dragons de bronze"
+
+
+ green_dragon:0 "Draconique Verte"
```

```suggestion
 green_dragon:0 "Dragon vert"
```
Je pense que "dragon vert" c'est suffisant ici.  Pareil pour les autres.